src/status.ts

```diff
@@ -6,12 +6,12 @@ import { NotificationBuilder } from './notification';
 
 export class MirrordStatus {
```

Comment:
What are the changes in this file for?

Reply:
Wanted to use the already existing constant for feedback command id, but the property was not static and we don't store any global instance of `MirrordStatus`. So I made all command ids static for consistency